docs/syncserver/README.md

```diff
@@ -0,0 +1,40 @@
+# Building and running Anki sync server in Docker
+
+This is an example Dockerfile contributed by an Anki user, which shows how we can run Self-Hosted Sync Server
+similar to what AnkiWeb.net offers
+
+Building and running Anki Sync Server within a container has the advantage of fully isolating
+the build products and runtime dependencies from the rest of your system.
+
+## Requirements
+
+- [x] [Docker](https://docs.docker.com/get-started/)
+
+# Building image
+
+By default docker buildx use the following version `ANKI_VERSION=23.2.1` by just running the below you should be able to build your own image:
```

Comment:
Users will forget to update the version. I'd recommend you make it a required input, and don't give it a default.